### DISCUSSION TITLE
Welcome to Image-Classification-using-Deep-Learning-for-a-city-dog-show Discussions!

### DISCUSSION THREAD

**Discussion by bhupali08** [bhupali08/Image-Classification-using-Deep-Learning-for-a-city-dog-show#10] [Announcements]

**Comment by SohamBera16** [comment]
1. Some of the ways to further optimize the model performance could be to tune the hyperparameters (using Grid Search or Bayesian Optimization based search) and use regularization methods like early stopping, dropout, etc. Also, data augmentation techniques could be explored for better performance.
